### PR TITLE
gearsystem 2.6.1

### PR DESCRIPTION
**Pull request by drhelius**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by igas** [comment]
```
==> /usr/local/opt/qt/bin/qmake PREFIX=/usr/local/Cellar/gearsystem/2.6.0 CONFIG+=c++11
/private/tmp/gearsystem-20190630-16054-1aio4vj/Gearsystem-gearsystem-2.6.0/platforms/macosx/Gearsystem/Gearsystem.pro:56: Extra characters after test expression.
Error processing project file: /private/tmp/gearsystem-20190630-16054-1aio4vj/Gearsystem-gearsystem-2.6.0/platforms/macosx/Gearsystem/Gearsystem.pro
```

**Comment by igas** [comment]
Also, as an author, could you provide some interface to test it? It would be great if we could ensure it built correctly. Please take a look at https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula, especially this part:

> We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. However, a bad test is better than no test at all.

**Comment by drhelius** [comment]
Sorry for the build failure, it's just a typo. Will fix that.

This is a Sega desktop emulator, it will be quite difficult to write a test to prove it's working. 

Will a test checking the application version using the command line be any good?

**Comment by igas** [comment]
It's not desirable, but it's better than nothing I guess. I'm not very familiar with that kind of software, but maybe some "validator" of the input file (game?) or something that could be stubbed. Just some ideas, maybe it'll help you to come up with something. Meanwhile, this looks ok to me 👌 Thank you for your contribution.

File: Formula/gearsystem.rb [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
-  desc "Sega Master System / Game Gear emulator"
+  desc "Sega Master System / Game Gear / SG-1000 emulator"
   homepage "https://github.com/drhelius/Gearsystem"
-  url "https://github.com/drhelius/Gearsystem/archive/gearsystem-2.5.1.tar.gz"
-  sha256 "bf834bddeb2a2d814229531eb302f64c0ea913af88cb7be508cac4b5e490790d"
+  url "https://github.com/drhelius/Gearsystem/archive/gearsystem-2.6.1.tar.gz"
+  sha256 "d7b0116e1a1cf6780bec4279e699dfbe9c4dd8d789f504aff1057b9fab079f47"
   head "https://github.com/drhelius/Gearsystem.git"
 
-  bottle do
```
Comment (igas) [inline Formula/gearsystem.rb]: You should keep this block, CI takes care of updating it.

**Comment by igas** [comment]
Thank you @drhelius!

**Pull request closed without merging**